mesheryctl/cmd/start.go

```diff
@@ -71,20 +71,29 @@ var startCmd = &cobra.Command{
 		for _, container := range containers {
 			if "/meshery_meshery_1" == container.Names[0] {
 				log.Info("Opening Meshery in your broswer. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery.")
-				ostype, err := exec.Command("uname", "-s").Output()
-				if err != nil {
-					log.Fatal("Unable to detect OS type. Warning message: \n", err)
-				}
-				os := strings.TrimSpace(string(ostype))
-				// Link to Meshery User Interface
-				url := "http://localhost:9081"
-				if os == "Linux" {
-					// Meshery running on Linux host
+				//ostype, err := exec.Command("uname", "-s").Output()
+				//if err != nil {
+				//	log.Fatal("Unable to detect OS type. Warning message: \n", err)
+				//}
+				//os := strings.TrimSpace(string(ostype))
+				//// Link to Meshery User Interface
+				//url := "http://localhost:9081"
+				//if os == "Linux" {
+				//	// Meshery running on Linux host
+				//	exec.Command("xdg-open", url).Start()
+				//} else {
+				//	// Asssume Meshery running on MacOS host
+				//	exec.Command("open", url).Start()
+				//}
+				if runtime.GOOS == "windows" {
```

Comment:
@anandrkskd it'd be good to keep the code comments in here for ease of understanding this logic. 

"Meshery running on a a Linux host."

Reply:
Yes sir it is in demo mode fir another issue, and I will do the comments.